conductr-bundle-lib/src/main/java/com/typesafe/conductr/bundlelib/LocationService.java

```diff
@@ -51,7 +51,8 @@ public static URL getLookupUrl(String serviceName, URL fallback) throws Malforme
     }
 
     static HttpPayload createLookupPayload(String serviceLocator, String serviceName) throws MalformedURLException {
-        URL locatorUrl = new URL(serviceLocator + serviceName);
+        String serviceNameWithLeadingSlash = serviceName.startsWith("/") ? serviceName : "/" + serviceName;
```

Comment:
`java-conductr-bundle-lib` which is the Java 8 bundle lib also uses the `com.typesafe.conductr.bundlelib.LocationService`. I've also added tests for `java-conductr-bundle-lib` to check if the service lookup works for service names with and without a leading slash.